src/test/java/com/amazon/ion/impl/IonRawTextReaderTest_1_1.java

```diff
@@ -126,4 +128,29 @@ public void invalidExpressionSyntax(int minorVersion, String input, String first
             assertThrows(IonReaderTextRawX.IonReaderTextParsingException.class, reader::nextRaw);
         }
     }
+
+    @ParameterizedTest
+    @CsvSource({
+        "(:values 0) (:values 1)",
+        "(:values (: values 0)) 1",
+        "(:values) 0 (:values) 1",
+        "(:values) (:values 0) 1",
+        "(:values) (:values) (:values 0) 1",
+        "(:values (:: ) ) 0 1",
+        "(:values (:: 0 1))",
+        "(:values 0 1)",
+        "(:values (:: (:: 0) (:values (:: 1))))",
+        "(:values (:: 0) (:values (:: 1)))",
+        "(:values (:values (:: 0 1)))",
+        "(:values (:values 0 1))"
```

Comment:
Does it support invoking my numeric address as well? (It's okay if that's not in scope for this PR, but if it is, we should add a test case for that too.)
```suggestion
        "(:values (:values 0 1))",
        "(:1 (:1 0 1))"
```

Reply:
Yes, that works. Added. There is coverage for invoking macros from text Ion by ID in `EncodingDirectiveCompilationTest` as well.